server/api/tumdev/campus_backend.proto

```diff
@@ -538,29 +537,35 @@ message Movie {
   string link = 16;
 }
 
-message SendFeedbackReply {}
 message SendFeedbackRequest {
-  string topic = 1;
-  string email = 2;
-  string email_id = 3;
-  string message = 4;
-  int32 image_count = 5;
-  double latitude = 6;
-  double longitude = 7;
-  string os_version = 8;
-  string app_version = 9;
-}
-
-message SendFeedbackImageReply {
-  string status = 1;
-}
+  message Metadata {
+    double latitude = 1;
+    double longitude = 2;
+    string osVersion = 3;
+    string appVersion = 4;
+  }
 
-message SendFeedbackImageRequest {
-  int32 id = 1;
-  int32 image_nr = 2;
```

Comment:
image_count might be a better name?

Reply:
This is not in the current API any more.
In the previous API this was nesseary to differtiate between images.
The current API does not use user provided names for the feedback anymore.